include/networkit/io/MTXGraphReader.hpp

```diff
@@ -0,0 +1,34 @@
+#ifndef NETWORKIT_IO_MTX_GRAPH_READER_HPP_
+#define NETWORKIT_IO_MTX_GRAPH_READER_HPP_
+
+#include <networkit/io/GraphReader.hpp>
+
+namespace NetworKit {
+
+/**
+ * @ingroup io
+ *
+ * Reader for the matrix market file format documented in
+ * https://networkrepository.com/mtx-matrix-market-format.html
+ *
+ * Does not allow complex fields.
+ *
+ * TODO: check possible combinations of field values e.g.:
```

Comment:
Is this TODO still valid?

Reply:
The code does not explicitly check for the examples given. I personally think, here the `garbage in, garbage out` rule applies: Checking for the sanity of certain graph conditions (possibly after parsing each line, or once after parsing the entire matrix) is very costly. Typically, the user knows which graphs they are using, and thereby, which conditions should be fulfilled. So making sure that the resulting graph satisfies those conditions, in my opinion, should be up to the user.

Reply:
I agree 👍 Then, in order to avoid confusion, please remove the `TODO` from the comments.